patch/1.19.9/nginx-upstream_mtls.patch

```diff
@@ -23,3 +23,227 @@ index 76045c4..cee3e2a 100644
      rc = ngx_ssl_handshake(c);
  
      if (rc == NGX_AGAIN) {
+@@ -2819,10 +2826,21 @@ ngx_http_upstream_process_headers(ngx_http_request_t *r, ngx_http_upstream_t *u)
+     ngx_http_upstream_main_conf_t  *umcf;
+ 
+     umcf = ngx_http_get_module_main_conf(r, ngx_http_upstream_module);
+-
++#if (NGX_HTTP_APISIX)
```

Comment:
Could you split this PR into two?
One is adding proxy ignore/hidden
Another is upgrading to Nginx 1.21.4

Reply:
I get it, please close this pr

Reply:
And the proxy ignore/hidden feature need to have its own patch, as it is not relative to mTLS.